lib/mfg.c

```diff
@@ -111,13 +114,30 @@ static void RSA_get0_key(const RSA *r, const BIGNUM **n,
 }
 #endif
 
+static void get_i2c_operands(enum switchtec_gen gen, uint32_t *addr_mask,
+			     uint32_t *map_shift, uint32_t *map_mask)
```

Comment:
This is a pretty ugly function and it's only used in one place.

Why not just inline the if and avoid the indirection.

Reply:
Yes it's not nice, but we need this later for 'config-set' as well. So let's keep it if it's OK.

Reply:
Hmm, ok, it's ugly, but I can't think of anything better.

But shouldn't the config-set be fixed up in the same patch?

Reply:
`config-set` has ~3 patches as well: different MRPC command, different I2C settings, different config file format. 
It might be too many commits for one PR. So this PR focuses on 'config-get' only, making it easier to review.

Reply:
I see what you mean. `config-set` is fixed as well, below.